src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -12269,8 +12269,55 @@ def output_col(col_name: ColumnOrName) -> SnowparkColumn:
         )
         return SnowflakeQueryCompiler(new_internal_frame)
 
-    def str_get(self, i: int) -> None:
-        ErrorMessage.method_not_implemented_error("get", "Series.str")
+    def str_get(self, i: int) -> "SnowflakeQueryCompiler":
+        """
+        Extract element from each component at specified position or with specified key.
+
+        Extract element from lists, tuples, dict, or strings in each element in the Series/Index.
```

Comment:
> lists, tuples, dict, or strings

This only supports extraction from string at the moment?

Reply:
That's right. Updated docs.